docs/_optimist/storage/localstorage/index.en.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Localstorage
+lang: "en"
+permalink: /optimist/storage/localstorage/
+nav_order: 3200
+parent: Storage
+has_children: false
+---
+
+# Compute localstorage for your instances
+
+## What exactly is Compute Localstorage?
+
+With Localstorage, the storage of your instances is located directly on the hypervisor (server). The localstorage feature (available through our l1 flavors) is intended for use with applications that require low latency.
+
+## Data security and availability
+
+Since your data is directly bound by your instance on our local hypervisor storage, we recommend that you distribute this data via a HA concept, over the provided Availability Zones.
+The storage backend of the local storage instances is protected against the failure of individual storage media in the array, however, the resulting redundancy compared to the Ceph-based instances only exists within the hypervisor node providing the instance.
+When replacing individual components following a hardware issue, there may be limited availability and performance for a short time until recovery.
+The hypervisors are subject to our defined patch cycle where we must boot through the hypervisors one by one. This will be performed per Availability Zone, one server at a time, within a defined Maintainace Window.
```

Comment:
This will be performed per Availability Zone, one server at a time, within a defined Maintainace Window.
We should remove it as it is doubled in the following sentence